Sandbox/src/Game/Player/Magic/Spell.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <Square.h>
+
+#include "Game/Items/Item.h"
+
+class Spell : public Square::GameObject
+{
+private:
+	int mIndex;
+	std::string mName;
+	int mLevelReq;
+	std::vector<Item*> mCastReq;
+	int mDamage;
+	int mBaseExperience;
+	Square::Sprite* mSprite;
+
+public:
+	Spell(int index, std::string name, int levelReq, std::vector<Item*> castReq, int damage, int baseExperience, std::string spritePath)
+	{
+		mIndex = index;
+		mName = name;
+		mLevelReq = levelReq;
+		mCastReq = castReq;
+		mDamage = damage;
+		mBaseExperience = baseExperience;
+
+		//mSprite = new Square::Sprite(spritePath);
```

Comment:
was this for testing? we can prob get rid of it since this is a part of the interface

Reply:
I had to comment out the sprites because they don't exist yet and throw an error at runtime. My next update will be a logging system for the engine so that problem can be fixed I guess